Docs/development/jasp-build-guide-windows.md

```diff
@@ -0,0 +1,149 @@
+
+```bash
+conan profile new default --detect
+conan profile show default
+```
+
+You should see something like this. CMake later uses this template to modify some of its parameters and download and prepare the dependencies accordingly. For instance, if you are building a Debug version of JASP, CMake will change this profile to account for that.
+
+```
+[settings]
+os=Windows
+os_build=Windows
+arch=x86_64
+arch_build=x86_64
+compiler=Visual Studio
+compiler.version=16
+build_type=Debug
+[options]
+[conf]
+[build_requires]
+[env]
+```
+
+> 💡 Although CMake and Qt Creator will run Conan process for you, if it's your very first time configuring JASP, and you ran into any problem, you can run the Conan command manually. If things go wrong, CMake configuration will stop and tells you what you should do to resolve the Conan issue. 
+
+### Configuring JASP Desktop
+
+At this point, you are ready to start configuring and building JASP. Open the Qt Creator, and select "File → Open Project", then find and open the `CMakeLists.txt` file inside the `jasp-desktop` folder. By opening this file, you are opening the entire JASP project, and and you will be prompted to "Manage Kits". Here you want to select the "Desktop Qt 6.2.3 MSVC2019 64bit" kit by checking the checkbox next to it. 
+
+> ⚠️ This is an important step, and if you miss selecting the **Desktop** kit, you will not be able to build JASp.
+
+After selecting your kit, you can select the "Create Configuration", and observe the Qt Creator starting to configure your project. You can see the progress of the CMake configuration in the "General Messages" output panel, usually Ctrl+6.
+
+If this is your first time preparing your project, CMake is going to configure **everything**, and as a result the *very first* configuration is going to be time confusing depending on your settings, internet speed, and computer performance. During the configuration, CMake downloads and instance of R, unpack it and place it in the appropriate location inside the build folder. In addition, it looks for all the necessary libraries and programs required as well. If any of the requirement isn't met, CMake will stop the configuration and notify you about the problem. However, if everything else is prepared correctly, you will not receive any errors, and your project will be configured, and you will see a set of messages as below:
+
+```
+-- Configuring done
+-- Generating done
+-- Build files have been written to: <path-to-your-build-folder>
+```
+
+#### ⚠️ R-Interface 
+
+CMake makes sure that it build the R-Interface using the MinGW x64 libraries every time (if necessary). So, unlike before, you don't need to worry about building it separately and copying it into the right place, however, you need to make sure that the `C:\msys64\mingw64\bin` is in your PATH. You can add this address to your Build Environment path inside the Qt Creator.
```

Comment:
You never had to "copy it to the right place", you had to set it up in a subdirectory of the normal buildfolder and it would place the dll next to `JASPEngine.exe`

Might be nitpicky though

Reply:
I modified it a bit, and we can remove it in a few months as we go.